frontend/src/lib/components/ActivityLog/activityLogLogic.insight.test.tsx

```diff
@@ -329,7 +329,7 @@ describe('the activity log logic', () => {
             const actual = logic.values.humanizedActivity
 
             expect(render(<>{actual[0].description}</>).container).toHaveTextContent(
-                'peter added tags 4 5 , and removed tags 2 3 on test insight'
+                'peter added tags 45, and removed tags 23 on test insight'
```

Comment:
Believe this is just the new dom structure. Checked the stories and things still render nicely
<img width="595" alt="Screenshot 2024-03-20 at 09 36 22" src="https://github.com/PostHog/posthog/assets/6685876/5dc5d95d-b8e5-41c4-9a5c-e4228eabe05e">